Split/FetcherEngine/Recorder/ImpressionsCountRecorderWorker.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ImpressionsCountRecorderWorker.swift
+//  Split
+//
+//  Created by Javier Avrudsky on 23/06/2021.
+//  Copyright © 2021 Split. All rights reserved.
+//
+
+import Foundation
+
+class ImpressionsCountRecorderWorker: RecorderWorker {
+
+    private let countsStorage: PersistentImpressionsCountStorage
+    private let countsRecorder: HttpImpressionsCountRecorder
+    private let kPopCount = ServiceConstants.defaultImpressionCountRowsPop
+
+    init(countsStorage: PersistentImpressionsCountStorage,
+         countsRecorder: HttpImpressionsCountRecorder) {
+        self.countsStorage = countsStorage
+        self.countsRecorder = countsRecorder
+    }
+
+    func flush() {
+        var rowCount = 0
+        var failedCounts = [ImpressionsCountPerFeature]()
+        repeat {
+            let counts = countsStorage.pop(count: kPopCount)
+            rowCount = counts.count
+            if rowCount > 0 {
+                Logger.d("Sending impressions")
```

Comment:
I would use `"Sending impressions counts"` to distinguish from ImpressionsRecorderWorker

Reply:
Fixed Doc!